protocol-specs/generators/lisk_codec/index.js

```diff
@@ -0,0 +1,555 @@
+		},
+	};
+
+
+	const numberEncoded32 = Number32.encode(input.message32.object).finish();
+	const signedNumberEncoded32 = SignedNumber32.encode(input.messageSigned32.object).finish();
+	const numberEncoded64 = Number64.encode(input.message64.object).finish();
+	const signedNumberEncoded64 = SignedNumber64.encode(input.messageSigned64.object).finish();
+
+	return {
+		description: 'Encoding of numeric types',
+		config: {
+			network: 'devnet',
+		},
+		input: {
+			message32: input.message32,
+			messageSigned32: input.messageSigned32,
+			message64: input.message64,
+			messageSigned64: input.messageSigned64,
+		},
+		output: {
+			numberEncoded32: numberEncoded32.toString('hex'),
+			signedNumberEncoded32: signedNumberEncoded32.toString('hex'),
+			numberEncoded64: numberEncoded64.toString('hex'),
+			signedNumberEncoded64: signedNumberEncoded64.toString('hex'),
+		},
+	};
+};
+
+const generateValidBooleanEncodings = () => {
+	const input = {
+		booleanTrue: {
+			object: {
+				state: true,
+			},
+			schema: {
+				type: 'object',
+				properties: {
+					state: {
+						dataType: 'boolean',
+						fieldNumber: 1,
+					},
+				},
+			},
+		},
+		booleanFalse: {
+			object: {
+				state: false,
+			},
+			schema: {
+				type: 'object',
+				properties: {
+					state: {
+						dataType: 'boolean',
+						fieldNumber: 1,
+					},
+				},
+			},
+		},
+	};
+
+	const booleanTrueEncoded = Boolean.encode(input.booleanTrue.object).finish();
+	const booleanFalseEncoded = Boolean.encode(input.booleanFalse.object).finish();
+
+	return {
+		description: 'Encoding of boolean types',
+		config: {
+			network: 'devnet',
+		},
+		input: {
+			booleanTrue: input.booleanTrue,
+			booleanFalse: input.booleanFalse,
+		},
+		output: {
+			booleanTrue: booleanTrueEncoded.toString('hex'),
+			booleanFalse: booleanFalseEncoded.toString('hex'),
+		},
+	};
+};
+
+const generateValidStringEncodings = () => {
+	const input = {
+		string: {
+			object: {
+				data: 'Checkout Lisk SDK!',
+			},
+			schema: {
+				type: 'object',
+				properties: {
+					data: {
+						dataType: 'string',
+						fieldNumber: 1,
+					},
+				},
+			},
+		},
+		emptyString: {
+			object: {
+				data: '',
+			},
+			schema: {
+				type: 'object',
+				properties: {
+					data: {
+						dataType: 'string',
+						fieldNumber: 1,
+					},
+				},
+			},
+		},
+	};
+
+	const stringEncoded = String.encode(input.string.object).finish();
+	const emptyStringEncoded = String.encode(input.emptyString.object).finish();
+
+	return {
+		description: 'Encoding of string types',
+		config: {
+			network: 'devnet',
+		},
+		input: {
+			string: input.string,
+			emptyString: input.emptyString,
+		},
+		output: {
+			string: stringEncoded.toString('hex'),
+			emptyString: emptyStringEncoded.toString('hex'),
+		},
+	};
+};
+
+
+const generateValidBytesEncodings = () => {
+	const input = {
+		bytes: {
+			object: {
+				address: Buffer.from('e11a11364738225813f86ea85214400e5db08d6e', 'hex').toString('hex'),
```

Comment:
The value `e11a11364738225813f86ea85214400e5db08d6e` is already a `hex` value. 

```suggestion
				address: 'e11a11364738225813f86ea85214400e5db08d6e',
```

Reply:
This was for simplifying the output as the default JSON.stringified version was not favoured by some reviewers. It has been corrected already.